fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/ExplainTokens.java

```diff
@@ -0,0 +1,1110 @@
+/*
+ * ExplainTokensWithPrecedence.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.query.plan.cascades;
```

Comment:
I'm not sure it makes sense for the new explain stuff to be in the `cascades` package, given that (other than arguably the symbol table stuff), it's not really particular to cascades plans. Either in the parent `plan` package or a new package (e.g., `com.apple.foundationdb.record.query.plan.explain`) would probably make more sense.

Reply:
I put all explain-related classes in `com.apple.foundationdb.record.query.plan.explain`